draft-ietf-quic-transport.md

```diff
@@ -3087,16 +3087,17 @@ state as a result of delayed delivery of packets.
 ### Bidirectional Stream States {#stream-bidi-states}
 
 A bidirectional stream is composed of a send stream and a receive stream.
-Implementations might present the state of the pair of streams as a composite of
-the states of the pair of streams.  The simplest model presents the stream as
-"open" when either send or receive stream is in a non-terminal state and
-"closed" when both send and receive streams are in a terminal state.
+Implementations might represent the state of the bidirectional stream as a
+composite of the states of the constituent send and receive streams.  The
```

Comment:
nits: 
I a prefer lower case 'may' to 'might' in the line above.
Should it states be plural?
I think constituent is unnecessary

Reply:
done